### PR TITLE
chore: Add configurable args to bdd test

### PR DESCRIPTION
**Pull request by fqutishat**
closes $532

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: test/bdd/didresolver_steps.go [hunk 1]
```diff
@@ -19,35 +19,40 @@ import (
...
-	"github.com/trustbloc/sidetree-core-go/pkg/document"
-	"github.com/trustbloc/sidetree-core-go/pkg/docutil"
-	"github.com/trustbloc/sidetree-node/models"
 )
 
-const sha2_256 = 18
-const didDocNamespace = "did:sidetree:"
-const maxRetry = 5
+const (
+	SideTreeURL     = "$SIDETREE_URL"
```
Comment (troyronda) [inline test/bdd/didresolver_steps.go]: You might want to do ${...}

File: test/bdd/bddtests_test.go [hunk 2]
```diff
@@ -95,10 +95,14 @@ func FeatureContext(s *godog.Suite) {
 		panic(fmt.Sprintf("Error returned from NewBDDContext: %s", err))
 	}
 
+	// set dynamic args
+	context.DynamicArgs[SideTreeURL] = "http://localhost:48326/.sidetree/document"
```
Comment (troyronda) [inline test/bdd/bddtests_test.go]: I don't think that `Dynamic` is adding to the name :)

Comment (fqutishat) [reply test/bdd/bddtests_test.go]: done

File: test/bdd/didresolver_steps.go (hunk 1 above)
Comment (fqutishat) [reply test/bdd/didresolver_steps.go]: done

**Pull request merged**